knowledge-base/peter-evans/find-comment/action-security.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Find Comment'
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
+    issues: read
```

Comment:
I think this needs `pull-requests: read` as well. otherwise PR comments will not be found.